en/docs/concepts/oauth2/intro-oauth2.md

```diff
@@ -28,17 +28,18 @@ These different grant types target different application types.
 | Flow                 | Usage         | 
 | --------------------- | ------------- | 
 | Authorization code grant | Mostly recommended for web applications and single-page applications(SPA). This method is very safe in web applications, as tokens are passed directly to the web server and stored there without exposing the tokens to the resource owners. For Single page applications (JavaScript clients) it is recommended to use this grant type with PKCE.  |                            
-| Client credential grant  | Recommended for machine to machine communications (CLIs), if a machine needs to access a resource without human interaction, this grant type is recommended to use.  |                              
-| Implicit grant            | Used for single-page applications (SPA) to retrieve the access tokens directly without spending a number of round trips. Anyway, there is a security consideration, as the access token is exposed to the client-side.  | 
+| Client credential grant  | Recommended for a machine to machine communications (CLIs), if a machine needs to access a resource without human interaction, this grant type is recommended to use.  |                              
```

Comment:
The "a" should not be here. 

this grant type is recommended to use --> "this grant type is recommended." or "it is recommended to use this grant type"

Reply:
different identities such as